src/com/walmartlabs/lacinia/pedestal/subscriptions.clj

```diff
@@ -98,7 +98,7 @@
         (async/timeout keep-alive-ms)
         (do
           (log/debug :event ::timeout)
-          (>! response-data-ch {:type :connection_keep_alive})
+          (>! response-data-ch {:type :ka})
```

Comment:
I splitted this PR and in #21 only the Apollo compatibility is fixed. Also tests pass